target/scripts/helpers/error.sh

```diff
@@ -2,7 +2,8 @@
 
 function _errex
 {
-  echo -e "Error :: ${*}\nAborting." >&2
+  _log 'error' "${1:?}"
```

Comment:
`:?` has nothing to with where the message it logged to. It makes sure the first argument is properly supplied. Otherwise `_log` would log an error that no message is provided, which would be misleading.

Reply:
> Otherwise _log would log an error that no message is provided, which would be misleading.

`_log` does check, if a message is provided or not:

```bash
  if [[ -z ${2+set} ]]
  then
    echo "Call to '_log' is missing a message to log" >&2
    return 1
  fi
```

This error message is more clear than `parameter null or not set` which `${1:?}` produces if  $1 is empty or not set.

Reply:
Hmm, true as well. I will double-check and take care!

Reply:
Done in 1cbde1d139f519716d0c439eb871fe7c37ded85d. I added an explicit message for better debugging.